angular/core/components/invitation_form/invitation_form.coffee

```diff
@@ -6,7 +6,7 @@ angular.module('loomioApp').factory 'InvitationForm', ->
       _.filter Session.user().groups(), (group) ->
         AbilityService.canAddMembers(group)
 
-    $scope.form = Records.invitationForms.build(groupId: group.id or (_.first($scope.availableGroups()) or {}).id)
+    $scope.form = Records.invitationForms.build(groupId: Session.currentGroupId() or (_.first($scope.availableGroups()) or {}).id)
```

Comment:
Maybe same thing here; if we ensure we're always passing in the currentGroup to the InvitationForm, this code will work as-is.